app/dashboard/views.py

```diff
@@ -639,6 +639,14 @@ def dashboard(request):
     }
     return TemplateResponse(request, 'dashboard/index.html', params)
 
+def ethhack(request):
+    """Handle displaying ethhack landing page."""
+
+    title = str(_(" Eth hackathon 2019"))
+    params = {
+        'title': title
```

Comment:
could we just use it as `'title':  _(" Eth hackathon 2019")`
That should suffice.

Reply:
fixed